StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostController.swift

```diff
@@ -10,12 +10,12 @@ import UIKit
 
 /// An internal result type that helps us handle both
 /// Financial Connections and Instant Debits
-@_spi(STP) public enum HostControllerResult {
+@_spi(STP) @frozen public enum HostControllerResult {
     case completed(Completed)
     case failed(error: Error)
     case canceled
 
-    @_spi(STP) public enum Completed {
+    @_spi(STP) @frozen public enum Completed {
```

Comment:
Froze these so that we wouldn't have to handle `@unknown default` when switching on the result.

If we ever do add more results, this will trigger a compile error in `StripeConnect` and ensure we think through what the expected behavior should be.

Reply:
this is no longer needed after changing to `presentForToken`